From: Michael Biebl <biebl@debian.org>
Date: Sat, 21 Oct 2023 20:09:20 +0200
Subject: Move D-Bus policy file to /usr/share/dbus-1/system.d/

To better support stateless systems with an empty /etc, the old location
in /etc/dbus-1/system.d/ should only be used for local admin changes.
Package provided D-Bus policy files are supposed to be installed in
/usr/share/dbus-1/system.d/.

This is supported since dbus 1.9.18.

https://lists.freedesktop.org/archives/dbus/2015-July/016746.html

https://gitlab.gnome.org/GNOME/NetworkManager-fortisslvpn/-/merge_requests/32
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index b2e5533..25efe0a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -21,7 +21,7 @@ SUBDIRS = \
 plugindir = $(libdir)/NetworkManager
 plugin_LTLIBRARIES =
 
-dbusservicedir = $(sysconfdir)/dbus-1/system.d
+dbusservicedir = $(datadir)/dbus-1/system.d
 dbusservice_DATA = nm-fortisslvpn-service.conf
 
 nmvpnservicedir = $(NM_VPN_SERVICE_DIR)
